dvc/stage.py

```diff
@@ -390,8 +391,9 @@ def _check_stage_path(repo, path):
         if not os.path.isdir(real_path):
             raise StagePathNotDirectoryError(path)
 
-        proj_dir = os.path.realpath(repo.root_dir) + os.path.sep
-        if not (real_path + os.path.sep).startswith(proj_dir):
+        proj_dir = os.path.realpath(repo.root_dir)
+
```

Comment:
Is this line necessary? seems to me that `proj_dir` is kinda related to next line

Reply:
Not necessary at all. Just tried to separate the logic. Will delete.

Reply:
@pared , what do you mean?
@skshetry , let's wait a little bit more, @pared  didn't requested any changes, it was just a comment)

Reply:
@mroutis, I now think, it was a bit unnecessary

Reply:
@mroutis I was talking about using newline, as @skshetry noted. It wouldn't be harmful, it just seemed to me that `proj_dir` is directly related to what is happening in the next line after it, so there is no reason to put newline here.

Reply:
ohhh got it, @pared ! don't know why I was thinking about the `proj_dir` variable itself.

Reply:
```suggestion
```